### PR TITLE
feat(languages): Elm

### PR DESCRIPTION
**Pull request by jaredramirez**
Hey there,

This PR adds support for [Elm](https://elm-lang.org/)!

### CODE REVIEW

**Comment by pickfire** [comment]
```
cargo xtask docgen
```

File: runtime/queries/elm/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+(value_expr name: (value_qid (upper_case_identifier) @label))
+(value_expr (upper_case_qid (upper_case_identifier) @label (dot) (upper_case_identifier) @variable.other.member))
+(value_expr(upper_case_qid(upper_case_identifier)) @variable.other.member)
+
+; comments
+(line_comment) @comment
+(block_comment) @comment
+
+; strings
+(string_escape) @constant.character
```
Comment (archseer) [inline runtime/queries/elm/highlights.scm:77]: `@constant.character.escape`

File: runtime/queries/elm/highlights.scm [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+(open_quote) @string
+(close_quote) @string
+(regular_string_part) @string
+
+(open_char) @constant.character
+(close_char) @constant.character
+
+
+; glsl
+(glsl_content) @source.glsl
```
Comment (archseer) [inline runtime/queries/elm/highlights.scm:88]: You can remove this, it's handled by injections

File: runtime/queries/elm/highlights.scm [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+"|" @keyword
+"," @punctuation.delimiter
+
+[
+  "|>"
+] @keyword
+
+
+(import) @keyword.other
```
Comment (archseer) [inline runtime/queries/elm/highlights.scm:53]: `@keyword.control.import`

File: runtime/queries/elm/highlights.scm [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+
+"(" @punctuation.braces
+")" @punctuation.braces
+
+"[" @punctuation.braces
+"]" @punctuation.braces
+
+"{" @punctuation.braces
+"}" @punctuation.braces
```
Comment (archseer) [inline runtime/queries/elm/highlights.scm:43]:
```suggestion
[
  "("
  ")"
  "["
  "]"
  "{"
  "}"
] @punctuation.bracket
```

**Review by archseer — CHANGES_REQUESTED** [review]

**Comment by kevinsjoberg** [comment]
This looks ready, right?

**Comment by jaredramirez** [comment]
Friendly ping @archseer 😃

**Review by the-mikedavis — APPROVED** [review]
this one lgtm too 🎉

**Pull request merged**